nengo_viz/static/viz_netgraph_item.js

```diff
@@ -36,7 +36,7 @@ VIZ.NetGraphItem = function(ng, info) {
     this.minHeight = 5;
 
     this.expanded = false;
-    
+
```

Comment:
I can select lines to commit? There is a change in this file (further down, lines 219, 226).